src/js/tabs/led_strip.js

```diff
@@ -394,19 +380,19 @@ TABS.led_strip.initialize = function (callback, scrollPosition) {
 
                     // set checkbox values
                     TABS.led_strip.overlays.forEach(function(letter) {
-                        var feature_o = $('.checkbox').find('input.function-' + letter);
+                        const featureO = $('.checkbox').find(`input.function-${letter}`);
 
-                        var newVal = ($(uiSelectedLast).is('.function-' + letter));
+                        const newVal = ($(uiSelectedLast).is(functionTag + letter));
 
-                        if (feature_o.is(':checked') != newVal) {
```

Comment:
What does the `O` in the name stand for?

Reply:
@McGiverGim do you know?

Reply:
I don't know... maybe overlay?